ui/components-react/src/components-react/filter-builder/FilterBuilderState.ts

```diff
@@ -110,15 +110,25 @@ export class PropertyFilterBuilderActions {
   }
 
   /** Adds new rule or group of rules to the group specified by path. */
-  public addItem(path: string[], itemType: "RULE_GROUP" | "RULE") {
+  public addItem(
+    path: string[],
+    itemType: "RULE_GROUP" | "RULE",
+    itemData?: PropertyFilterBuilderRule | PropertyFilterBuilderRuleGroup
```

Comment:
```suggestion
    item: "RULE_GROUP" | "RULE" | PropertyFilterRule | PropertyFilterRuleGroup,
```

I think `addItem` should accept `PropertyFilterRule/Group` and create `id`s inside.

Reply:
Added